geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -17,55 +17,54 @@ See the License for the specific language governing permissions and
 
-The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis
-commands.
+The <%=vars.product_name%> cluster must have at least one server that is set up to handle the incoming Redis commands.
 
 Use gfsh to start at least one server with a command of the form:
 
 ```pre
 start server \
   --name=<serverName> \
   --locators=<locatorPort> \
-  --redis-port=<redisPort> \
-  --redis-bind-address=<redisBindAddress> \
-  --redis-password=<redisPassword>
+  --compatible-with-redis-port=<compatibleWithRedisPort> \
+  --compatible-with-redis-bind-address=<compatibleWithRedisBindAddress> \
+  --compatible-with-redis-password=<compatibleWithRedisPassword>
 ```
 
-If any of the options `redis-bind-address`, `redis-password`, or `redis-port` are included, a compatible-with-Redis Geode server will be started.
+If any of the options `compatible-with-redis-bind-address`, `compatible-with-redis-password`, or `compatible-with-redis-port` are included, a Geode server compatible with Redis will be started.
 
 Replace `<serverName>` with the name of your server.
 
 Replace `<locatorPort>` with your locator port.
 
-Replace `<redisPort>` with the port that the <%=vars.product_name%> server listens on for Redis commands. The typical port used with a Redis cluster is 6379.
+Replace `<compatibleWithRedisPort>` with the port that the <%=vars.product_name%> server listens on for Redis commands. The typical port used with a cluster compatible with Redis is 6379.
 
-Replace `<redisBindAddress>` with the address of the server host.
+Replace `<compatibleWithRedisBindAddress>` with the address of the server host.
 
-Replace `<redisPassword>` with the password clients use to authenticate.
+Replace `<compatibleWithWithRedisPassword>` with the password clients use to authenticate.
 
 To confirm the server is listening, run:
 
 ``` pre
-redis-cli -h <redisBindAddress> -p <redisPort> -a <redisPassword> ping
+redis-cli -h <compatibleWithRedisBindAddress> -p <compatibleWithRedisPort> -a <compatibleWithRedisPassword> ping
 ```
 
-Replace `<redisBindAddress>`,`<redisPort>`, and `<redisPassword>` with the same values as the server.
+Replace `<compatibleWithRedisBindAddress>`,`<compatibleWithRedisPort>`, and `<compatibleWithRedisPassword>` with the same values as the server.
 
 If the server is functioning properly, you should see a response of `PONG`.
 
 ## <a id="included-commands" class="no-quick-link"></a>Included Redis Commands
 
-The Redis API for <%=vars.product_name%> currently supports the following commands.  See [Redis commands](https://redis.io/commands/) for a complete list of and more information on Redis commands.
+The <%=vars.product_name%> API for Compatibility with Redis currently supports the following Redis commands.  See [Redis commands](https://redis.io/commands/) for a complete list of and more information on Redis commands.
```

Comment:
The <%=vars.product_name%> APIs compatible with Redis currently supports the following Redis commands.